zkvm/tests/zkvm.rs

```diff
@@ -4,6 +4,79 @@ use hex;
 
 use zkvm::*;
 
+trait ProgramHelper {
+    fn issue_helper(
+        &mut self,
+        qty: u64,
+        flv: Scalar,
+        issuance_pred: Predicate,
+        nonce_pred: Predicate,
+    ) -> &mut Self;
+
+    fn input_helper(&mut self, qty: u64, flv: Scalar, pred: Predicate) -> &mut Self;
+
+    fn cloak_helper(&mut self, input_count: usize, outputs: Vec<(u64, Scalar)>) -> &mut Self;
+
+    fn output_helper(&mut self, pred: Predicate) -> &mut Self;
+}
+
+impl ProgramHelper for Program {
```

Comment:
_Extension traits_ are called so because you can implement the methods _inside the trait_, so the user can attach the trait w/o implementing anything: `impl ExtTrait for MyThing {}`. Here it works either way, but it's cleaner to declare the methods inside the trait to show that the trait is not the abstract interface, but simply a collection of add-on functionality.